cosmoz-treenode-navigator.js

```diff
@@ -67,7 +63,7 @@
 
 			chosenNode: {
 				type: Object,
-				computed: '_getNode(value, data)',
+				computed: '_getNode(value)',
```

Comment:
The `_getNode` function needs to be bound to the tree, so that it will be called only when the tree is available.